src/test/java/com/spotify/autoscaler/AutoscaleJobTest.java

```diff
@@ -264,17 +264,17 @@ public void testThatWeSampleAShortPeriodIfWeHaveChangedSizeRecently() {
 
   @Test
   public void testWeResizeIfSizeConstraintsAreNotMet() throws IOException {
-    final int loadDelta = 10;
+    final int minNodesOverride = 10;
     final BigtableCluster cluster =
         BigtableClusterBuilder.from(this.cluster)
-            .loadDelta(loadDelta)
+            .minNodesOverride(minNodesOverride)
             .lastChange(Instant.now())
             .build();
     AutoscaleJobTestMocks.setCurrentSize(bigtableInstanceClient, MIN_NODES);
     job = new AutoscaleJob(stackdriverClient, db, autoscalerMetrics);
     AutoscaleJobTestMocks.setCurrentLoad(stackdriverClient, 0.1);
     job.run(cluster, bigtableSession, Instant::now);
-    assertEquals(Optional.of(MIN_NODES + loadDelta), newSize);
+    assertEquals(Optional.of(Math.max(MIN_NODES, minNodesOverride)), newSize);
```

Comment:
The assertion should be 
`assertEquals(Optional.of(minNodesOverride), newSize);` to avoid testing the wrong thing in the future. It aims to test we do resize if we have a new min nodes override even id we had just resized. and to be completely safe and clear i'd also change to `final int minNodesOverride = MIN_NODES +10;`.